src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseHeaderLabel.cs

```diff
@@ -3,33 +3,55 @@
 
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
-using System.Runtime.Versioning;
 
 namespace System.Security.Cryptography.Cose
 {
+    /// <summary>
+    /// Represents a COSE header label.
+    /// </summary>
     [DebuggerDisplay("{DebuggerDisplay,nq}")]
     public readonly struct CoseHeaderLabel : IEquatable<CoseHeaderLabel>
     {
         internal string LabelName => LabelAsString != null ? $"\"{LabelAsString}\"" : LabelAsInt32.ToString();
         private string DebuggerDisplay => $"Label = {LabelName}, Type = {(LabelAsString != null ? typeof(string) : typeof(int))}";
 
         // https://www.iana.org/assignments/cose/cose.xhtml#header-parameters
+        /// <summary>
+        /// Gets a <see cref="CoseHeaderLabel"/> that represents the known header parameter "alg".
```

Comment:
The type name should not be said in a property summary, per https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Property.  (`Gets or sets <summary without specifying the type>.` or `Gets <summary without specifying the type>.`)

So this would be 

```suggestion
        /// Gets a value that represents the known header parameter "alg".
```

Reply:
Shouldn't it be "Gets a label"? 
```suggestion
        /// Gets a label that represents the known header parameter "alg".
```

Reply:
Yeah, "label" seems better here.